sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/_operations/_patch.py

```diff
@@ -7,8 +7,168 @@
+
+
+
+
+
+
+class RadiologyInsightsClientOperationsMixin(GeneratedRadiologyInsightsClientOperationsMixin):
+
+    @overload
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: _models.RadiologyInsightsJob,
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:...
+        
+
+    @overload
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: JSON,
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:...
+        
+
+    @overload
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: IO[bytes],
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:...
+        
+
+
+    @distributed_trace
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: Union[_models.RadiologyInsightsJob, JSON, IO[bytes]],
+        *,
+        expand: Optional[List[str]] = None,
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:
+        
+        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+        _params = kwargs.pop("params", {}) or {}
+
+        content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
+        cls: ClsType[_models.RadiologyInsightsJob] = kwargs.pop("cls", None)
+        polling: Union[bool, PollingMethod] = kwargs.pop("polling", True)
+        lro_delay = kwargs.pop("polling_interval", self._config.polling_interval)
+        cont_token: Optional[str] = kwargs.pop("continuation_token", None)
+        if cont_token is None:
+            raw_result = self._infer_radiology_insights_initial(
+                id=id,
+                resource=resource,
+                expand=expand,
+                content_type=content_type,
+                cls=lambda x, y, z: x,
+                headers=_headers,
+                params=_params,
+                **kwargs
+            )
+            raw_result.http_response.read() # type: ignore
+        kwargs.pop("error_map", None)
+
+        def get_long_running_output(pipeline_response):
+            response_headers = {}
+            response = pipeline_response.http_response
+            response_headers['x-ms-request-id']=self._deserialize('str', response.headers.get('x-ms-request-id'))
+            response_headers['Operation-Location']=self._deserialize('str', response.headers.get('Operation-Location'))
+            
+            deserialized = _deserialize(
+                _models.RadiologyInsightsInferenceResult,
```

Comment:
Shouldnt this stay as RadiologyInsightsJob? Seems like we're doing a get for the result property on that model.

Reply:
I ran the post processing script on your changes and I managed to run the tests on your branch successfully. Just run the post processing script again and push the changes it makes.